spec/formatting.md

```diff
@@ -692,10 +698,11 @@ and an `en` (English) locale,
 the pattern selection proceeds as follows for this message:
 
 ```
-.match {$count :number}
-one {{Category match}}
-1   {{Exact match}}
-*   {{Other match}}
+.input {$count :number}
+.match {$count}
+one {{Category match for {$count}}}
+1   {{Exact match for {$count}}}
+*   {{Other match for {$count}}}
```

Comment:
Is the goal of this change to illustrate the use of a declaration with a selector? Otherwise the change here isn't necessary.

Reply:
I thought that having at least one of the selection examples use a declaration would be good.